src/DurableWorker/DurableController.cs

```diff
@@ -88,46 +94,22 @@ public bool TryGetInputBindingParameterValue(string bindingName, out object valu
 
         public void AddPipelineOutputIfNecessary(Collection<object> pipelineItems, Hashtable result)
         {
-            var shouldAddPipelineOutput =
-                _durableFunctionInfo.Type == DurableFunctionType.ActivityFunction;
-
-            if (shouldAddPipelineOutput)
+ 
+            if (ShouldSuppressPipelineTraces())
             {
                 var returnValue = FunctionReturnValueBuilder.CreateReturnValueFromFunctionOutput(pipelineItems);
                 result.Add(AzFunctionInfo.DollarReturn, returnValue);
             }
         }
 
-        public bool TryInvokeOrchestrationFunction(out Hashtable result)
+        public Hashtable InvokeOrchestrationFunction()
         {
-            if (!_durableFunctionInfo.IsOrchestrationFunction)
-            {
-                result = null;
-                return false;
-            }
-
-            result = _orchestrationInvoker.Invoke(_orchestrationBindingInfo, _powerShellServices);
-            return true;
+            return _orchestrationInvoker.Invoke(_orchestrationBindingInfo, _powerShellServices);
         }
 
         public bool ShouldSuppressPipelineTraces()
         {
             return _durableFunctionInfo.Type == DurableFunctionType.ActivityFunction;
         }
-
-        private static OrchestrationBindingInfo CreateOrchestrationBindingInfo(IList<ParameterBinding> inputData)
-        {
-            // Quote from https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-bindings:
-            //
-            // "Orchestrator functions should never use any input or output bindings other than the orchestration trigger binding.
-            //  Doing so has the potential to cause problems with the Durable Task extension because those bindings may not obey the single-threading and I/O rules."
-            //
-            // Therefore, it's by design that input data contains only one item, which is the metadata of the orchestration context.
-            var context = inputData[0];
-
-            return new OrchestrationBindingInfo(
-                context.Name,
-                JsonConvert.DeserializeObject<OrchestrationContext>(context.Data.String));
-        }
```

Comment:
this method is removed because it's logic has been spread out to account for the external SDK